src/routing/Routes.tsx

```diff
@@ -3,16 +3,16 @@ import { Switch, Route, Redirect } from 'react-router-dom'
 import { Responsive } from 'decentraland-ui'
 import { env } from 'decentraland-commons'
 import Intercom from 'decentraland-dapps/dist/components/Intercom'
+import { default as DappApp } from 'decentraland-dapps/dist/containers/App'
```

Comment:
Couldn't this just be `App` instead of `{ default as DappApp }`?

Reply:
yus